test/t8_IO/t8_gtest_vtk_reader.cxx

```diff
@@ -44,7 +44,7 @@ class vtk_reader: public testing::TestWithParam<std::tuple<int, int, int>> {
     SC_CHECK_MPI (mpiret);
     mpiret = sc_MPI_Comm_rank (sc_MPI_COMM_WORLD, &mpirank);
     SC_CHECK_MPI (mpiret);
-    if (mpisize > T8_VTK_TEST_NUM_PROCS) {
+    if (mpisize != T8_VTK_TEST_NUM_PROCS) {
```

Comment:
Could you add a comment on why only mpisize = T8_VTK_TEST_NUM_PROCS is allowed?
Something along the lines of
`/** mpisize must fit to the number of files that are read in */`

Reply:
Done!